x-pack/test_serverless/api_integration/test_suites/observability/threshold_rule/group_by_fired.ts

```diff
@@ -0,0 +1,189 @@
+              },
+            ],
+            alertOnNoData: true,
+            alertOnGroupDisappear: true,
+            searchConfiguration: {
+              query: {
+                query: '',
+                language: 'kuery',
+              },
+              index: DATA_VIEW_ID,
+            },
+            groupBy: ['host.name', 'labels.groupId'],
+          },
+          actions: [
+            {
+              group: FIRED_ACTIONS_ID,
+              id: actionId,
+              params: {
+                documents: [
+                  {
+                    ruleType: '{{rule.type}}',
+                  },
+                ],
+              },
+              frequency: {
+                notify_when: 'onActionGroupChange',
+                throttle: null,
+                summary: false,
+              },
+            },
+          ],
+        });
+        ruleId = createdRule.id;
+        expect(ruleId).not.to.be(undefined);
+      });
+
+      it('should be active', async () => {
+        const executionStatus = await waitForRuleStatus({
+          id: ruleId,
+          expectedStatus: 'active',
+          supertest,
+        });
+        expect(executionStatus.status).to.be('active');
+      });
+
+      it('should set correct information in the alert document', async () => {
+        const resp = await waitForAlertInIndex({
+          esClient,
+          indexName: THRESHOLD_RULE_ALERT_INDEX,
+          ruleId,
+        });
+
+        expect(resp.hits.hits[0]._source).property(
+          'kibana.alert.rule.category',
+          'Threshold (Technical Preview)'
+        );
+        expect(resp.hits.hits[0]._source).property('kibana.alert.rule.consumer', 'alerts');
+        expect(resp.hits.hits[0]._source).property('kibana.alert.rule.name', 'Threshold rule');
+        expect(resp.hits.hits[0]._source).property('kibana.alert.rule.producer', 'observability');
+        expect(resp.hits.hits[0]._source).property('kibana.alert.rule.revision', 0);
+        expect(resp.hits.hits[0]._source).property(
+          'kibana.alert.rule.rule_type_id',
+          'observability.rules.threshold'
+        );
+        expect(resp.hits.hits[0]._source).property('kibana.alert.rule.uuid', ruleId);
+        expect(resp.hits.hits[0]._source).property('kibana.space_ids').contain('default');
+        expect(resp.hits.hits[0]._source)
+          .property('kibana.alert.rule.tags')
+          .contain('observability');
+        expect(resp.hits.hits[0]._source).property('kibana.alert.action_group', 'threshold.fired');
+        expect(resp.hits.hits[0]._source).property('tags').contain('observability');
+        expect(resp.hits.hits[0]._source).property('kibana.alert.instance.id', 'host-0,group-0');
+        expect(resp.hits.hits[0]._source).property('kibana.alert.workflow_status', 'open');
+        expect(resp.hits.hits[0]._source).property('event.kind', 'signal');
+        expect(resp.hits.hits[0]._source).property('event.action', 'open');
```

Comment:
We should add a check for `host.name` in this case. If it is the same logic as MTR, `host.name` would be indexed to alert document.

Reply:
Added [here](https://github.com/elastic/kibana/pull/161569/commits/88adeadb0d3dddf78fbfdfcf4e634e1d85112d53) 🙂